cinch/roles/jenkins_master/templates/etc/nginx/conf.d/jenkins_http.conf

```diff
@@ -19,7 +22,11 @@ server {
     proxy_set_header        X-Forwarded-For $proxy_add_x_forwarded_for;
     proxy_set_header        X-Forwarded-Proto $scheme;
     proxy_pass              http://jenkins;
-  }
+    limit_req zone=jenkins burst=100 nodelay;
```

Comment:
What does this 'burst=100' number mean? Is this something that should be parameterized?

Reply:
burst gives a little 'padding' by putting requests into a queue if one hits a limit. So here adding an additional 100 slot queue to lessen the impact of being rate limited. Seems to be recommended on nginx.com's blog and docs. Maybe we can parameterize this one.

Reply:
I don't see a downside to parameterizing all the number values. It gives users an opportunity to tweak those values if needed, and even if they are never changed by anyone, having the values documented with comments in our vars files (with links to specific nginx docs) makes it easier for cinch users to understand what's going on.